api/proto/buf.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+name: buf.build/mobilecoin/api
+lint:
+  use:
+    - DEFAULT
```

Comment:
nit: most likely should be a follow on if any change is desired.

Guessing these files came form `buf mod init` and then stripping out the comments.
having lint here implies that linting is meant to happen, but it looks like all the proto files have lint warnings.  While removing the lint key doesn't disable linting, perhaps it would indicate to maintainers that these don't fully support linting yet.